src/tests/integration/test-data/foundry-show

```diff
@@ -8,5 +8,4 @@ AssertTest.test_revert_branch(uint256,uint256)
 AssumeTest.test_assume_false(uint256,uint256)
 AssumeTest.testFail_assume_false(uint256,uint256)
 AssumeTest.testFail_assume_true(uint256,uint256)
-LoopsTest.sum_N(uint256)
```

Comment:
This test wasn't actually being run on the legacy backend (because it's in the `foundry-prove-skip-legacy`), so it's deleted here and its output removed.